hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -1,13 +1,55 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 
+import boto3
 import json
 from datetime import datetime
+import os
+from dataclasses import dataclass
+from functools import lru_cache
+
+dynamodb = boto3.resource("dynamodb")
+
+@dataclass
+class FetcherConfig:
+
+    output_s3_bucket: str
+    collab_config_table: str
+
+    @classmethod
+    @lru_cache(maxsize=1)  # probably overkill, but at least it's consistent
+    def get(cls):
+        return cls(
+            output_s3_bucket=os.environ["THREAT_EXCHANGE_DATA_BUCKET_NAME"],
+            collab_config_table=os.environ["THREAT_EXCHANGE_CONFIG_DYNAMODB"]
+        )
+
 
 def lambda_handler(event, context):
+    config = FetcherConfig.get()
+
+    paginator = dynamodb.meta.client.get_paginator('scan')
+
+    response_iterator = paginator.paginate(
+        TableName=config.collab_config_table,
+        ProjectionExpression=",".join(("#name", "privacy_group", "tags")),
+        ExpressionAttributeNames={"#name": "name"}
+    )
```

Comment:
nit: Should we move all this into the FetcherConfig class? Something like `collab_iterator = FetcherConfing.getIterator()` or even `collabs = FetcherConfing.getCollabs()`